python/tests/orderby_tests/orderby_int.py

```diff
@@ -0,0 +1,111 @@
+from tests.aggregate_tests.aggtst_base import TstView
+
+
+class orderby_asc_int(TstView):
+    def __init__(self):
+        # Validated on SQLite
```

Comment:
These are by definition validated on SQLite, so perhaps you can delete this comment everywhere.
Can you also delete the self.data = []? It's not really useful (but it may cause problems, so try one first)